Day-II-Calculator/calculator/src/components/DisplayComponents/CalculatorDisplay.js

```diff
@@ -1,2 +1,9 @@
 import React from 'react';
-import './Display.css';
+
+function CalculatorDisplay(props) {
+  return (
+    <div className="display">{props.value}</div>
```

Comment:
You could make this slightly easier on yourself with [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring):
```javascript
function CalculatorDisplay({ value }) {
  return <div className="display">{value}</div>;
}
```